src/styles.scss

```diff
@@ -15,12 +15,35 @@ body {
   font-family: "Work Sans", sans-serif;
   color: $black-color-400;
   line-height: normal;
+  width: calc(100% - 0.475rem);
+  position: relative;
+  overflow-y: scroll;
+}
+
+::-webkit-scrollbar {
```

Comment:
**[This feature is non-standard and is not on a standards track.](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar)**